tools/autograd/derivatives.yaml

```diff
@@ -278,6 +278,12 @@
 - name: any.dim(Tensor self, int dim, bool keepdim=False) -> Tensor
   output_differentiability: [False]
 
+- name: _is_all_true(Tensor self) -> Tensor
+  self: non_differentiable
+
+- name: _test_check_tensor(Tensor self) -> Tensor
+  self: non_differentiable
+
```

Comment:
This doesn't need a gradient formula, right? (I would expect adding a gradient formula to make it CompositeExplicitAutograd, but... I'm not familiar with the codegen for this).